wiki/Game_Modifiers/en.md

```diff
@@ -1,297 +1,333 @@
 
-In [Taiko](Taiko "wikilink") mode, the speed is raised and the overall difficulty increased just like in osu! mode. Since the hit leniency is greatly increased on top of Taiko's already strict overall difficulty in addition to making slider ticks a lot harder to hit and reducing the value of spinners, Double Time is usually regarded as the hardest mod in Taiko, and is seldom used.
+In [osu!taiko](/wiki/Game_Modes/osu!taiko) mode, the speed is raised and the overall difficulty increased just like in osu!standard. Since the hit leniency is greatly increased on top of osu!taiko's already strict overall difficulty in addition to making slider ticks a lot harder to hit and reducing the value of spinners, Double Time is usually regarded as the hardest mod in osu!taiko, and is seldom used.
 
-In [Catch the Beat](Catch_the_Beat "wikilink") mode, the speed and approach rate are changed the same as in other modes. However, since Catch the Beat has no overall difficulty to increase, this mod does nothing more than a BPM and approach rate multiplier and is thus only worth a 1.06x multiplier. Being the opposite of Half Time, though, Double Time greatly decreases the hyperdash leniency, making fruits requiring hyperdashes nigh impossible to catch in some cases.
+In [osu!catch](/wiki/Game_Modes/osu!catch) mode, the speed and approach rate are changed the same as in other modes. However, since Catch the Beat has no overall difficulty to increase, this mod does nothing more than a BPM and approach rate multiplier and is thus only worth a 1.06x multiplier. Being the opposite of Half Time, though, Double Time greatly decreases the hyperdash leniency, making fruits requiring hyperdashes nigh impossible to catch in some cases.
 
-#### <img src="Nightcore.png" title="fig:Nightcore.png" alt="Nightcore.png" width="69" height="65" /> Nightcore
+#### Nightcore
 
-***Click on Double Time again to activate Nightcore mod***
+![Nightcore mod icon](/wiki/shared/Nightcore.png "Nightcore mod icon")
 
-***"uguuuuuuuu"***
+Click on Double Time again to activate Nightcore mod
 
-If you click the Double Time icon again, it becomes the Nightcore mod which increases the pitch and adds a bumpin' bass beat to the background. The multiplier stays the same, and the beatmap itself is unaffected beyond what Double Time would do because <u>**Double Time will automatically be activated in order to use Nightcore mod**.</u>
+**Description**: uguuuuuuuu
 
-[This mod was originated from an April Fool joke.](https://osu.ppy.sh/forum/t/49733)
+If you click the Double Time icon again, it becomes the Nightcore mod which increases the pitch and adds a drum track to the background that (in 4/4 time) goes in the order of bass drum on beats 1 & 3, clap on beats 2 & 4, and crash on beat 1 every 4 measures. The multiplier stays the same, and the beatmap itself is unaffected beyond what Double Time would do because **Double Time will automatically be activated in order to use Nightcore mod**.
```

Comment:
You could optionally add that in 6/8 (or 3/4) time, the bass beat goes like this:
- Bass drum on beats 1 and 4
- Clap on beats 3 and 6 
- Crash every 4 measures